Menu/LobbySelectMenu.cs

```diff
@@ -213,6 +213,12 @@ public override void Update()
 
         private void Play(SimplerButton obj)
         {
+            if (ModManager.JollyCoop)
+            {
+                ShowErrorDialog("Please disable JollyCoop before playing Online");
```

Comment:
@Chrometurtle I added a check to ensure that Jolly was disabled before one could press Play to protect against unexpected behavior. Thoughts? I can push in separate PR if we want to keep RegionGate activity clean.

Reply:
@henpemaz :D?